execution_engine/src/storage/trie_store/operations/tests/mod.rs

```diff
@@ -129,13 +132,27 @@ impl ToBytes for TestValue {
     }
 }
 
+// Determine if a there exists a caller in the backtrace that matches any of the specified symbols
```

Comment:
Nit:
```suggestion
// Determine if there exists a caller in the backtrace that matches any of the specified symbols
```

Reply:
Fixed in https://github.com/casper-network/casper-node/pull/3984/commits/60be67bc699821fc9b2f664f05a8ac6db2b057d2